src/widgets/attachments/attachments-overview.component.tsx

```diff
@@ -33,7 +34,8 @@ export default function AttachmentsOverview() {
             thumbnailWidth: 320,
             thumbnailHeight: 212,
             caption: attachment.comment,
-            isSelected: false
+            isSelected: false,
+            dateTime: `${new Date(attachment.dateTime)}`
```

Comment:
What's the difference between `new Date(isoDateHere).toISOString()` and just `isoDateHere`?

Reply:
@FlorianRappl not sure I understand the question. But the reason I created a `Date` object from the `dateTime` value is because the `dateTime` value was not very readable. This is a sample `2020-04-20T13:18:57.000+0100`.

Reply:
When I do

```js
`${new Date()}`
// "Thu Jul 09 2020 08:06:41 GMT+0200 (Central European Summer Time)"
```

it is not so readable either. Furthermore, it presents a potential problem with the chosen locale. Could we use something like `dayjs` (used anyway in the lib) to format the date properly? Many thanks!

Reply:
> Could we use something like dayjs (used anyway in the lib) to format the date properly? 

I couldn't find any reference to `dayjs` in the [library](https://github.com/benhowell/react-grid-gallery/search?q=DayJS&unscoped_q=dayjs). And it seems it's not currently been used in the widgets repo either. I wanted to confirm that I understand what you meant before I introduce a new dependency.

Reply:
Maybe I'm blind:
https://github.com/openmrs/openmrs-esm-patient-chart-widgets/blob/master/package.json#L87

Just use this one - we use it in other microfrontends, too, for formatting and computations on date and time objects.

And sorry for the confusion, when I wrote "used anyway in the lib" I mean of course "used anyway in the current package", where lib = current package = openmrs-patient-chart-widgets. I thought that was clear. My mistake.

Reply:
@FlorianRappl I'm not sure how I missed this. Sorry about that. Please let me know if this is ok now. Visually it looks better than before.